homeassistant/components/zwave_js/api.py

```diff
@@ -1189,23 +1168,20 @@ def device_registered(device: DeviceEntry) -> None:
 @websocket_api.websocket_command(
     {
         vol.Required(TYPE): "zwave_js/remove_failed_node",
-        vol.Required(ENTRY_ID): str,
-        vol.Required(NODE_ID): int,
+        vol.Required(DEVICE_ID): str,
     }
 )
 @websocket_api.async_response
 @async_handle_failed_command
-@async_get_entry
+@async_get_node
```

Comment:
Can we assume that failed nodes are present in the controller list?

Reply:
Ok. So as long as we keep this command exposed in the device page, instead of eg on the network level, it should be safe. :+1: